connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/webhook/WebhookConnectorRegistry.java

```diff
@@ -16,16 +16,29 @@
  */
 package io.camunda.connector.runtime.inbound.webhook;
 
+import io.camunda.connector.api.inbound.Activity;
+import io.camunda.connector.api.inbound.Severity;
 import io.camunda.connector.runtime.inbound.lifecycle.ActiveInboundConnector;
 import io.camunda.connector.runtime.inbound.webhook.model.CommonWebhookProperties;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Optional;
+import java.util.regex.Pattern;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class WebhookConnectorRegistry {
 
+  private static final String WEBHOOK_CONNECTOR_REGISTRY = "WebhookConnectorRegistry";
+
+  // Reflect changes to this pattern in webhook element templates
+  private static final Pattern CURRENT_WEBHOOK_PATH_PATTERN =
+      Pattern.compile("^[a-zA-Z0-9]([a-zA-Z0-9_-]*[a-zA-Z0-9])?$");
```

Comment:
This allows paths like this: ```123__-23131```. Would be great to only allow hyphens and underscores between alphanumeric sections like this: `123_456-789`

Reply:
Sure!

Let me challenge some of my other assumptions about the desired pattern:

Should each "block" between hyphens and underscores should be exclusively numbers or exclusively letters? e.g.,
Valid: 123456-abcdef
Invalid: 1a2b3c-4d5e6f

Should we only allow lower case letters? e.g.,
Valid: abcdef
Invalid: ABCdef, ABCDEF

These might seem like silly questions, but I'm trying to build up my intuition on what's required here 😄 I'll update the pattern once I feel I understand the requirements.

Reply:
Its really up to us. We should support upper + lower case from my perspective.

Reply:
Ok! In my opinion, the only new restriction we should add is:

> only allow hyphens and underscores between alphanumeric sections like this: 123_456-789

I believe adding this restriction this would result in the pattern:
`^[a-zA-Z0-9]([a-zA-Z0-9]*[a-zA-Z0-9_-])?([a-zA-Z0-9_-][a-zA-Z0-9]*)*`

I'll update the code and unit tests and push in a few minutes.

Reply:
I pushed the updated code, unit tests, and element templates.

@sbuettner let me know what you think

Reply:
Looks good! Thanks for taking care.